android/src/main/kotlin/com/wire/kalium/presentation/MainActivity.kt

```diff
@@ -82,7 +82,7 @@ class MainActivity : ComponentActivity() {
     }
 
     private suspend fun provideAuthScope(coreLogic: CoreLogic, backendLinks: ServerConfig.Links): AuthenticationScope =
-        when (val result = coreLogic.versionedAuthenticationScope(backendLinks).invoke()) {
+        when (val result = coreLogic.versionedAuthenticationScope(backendLinks).invoke(AutoVersionAuthScopeUseCase.ProxyCredentials.None)) {
```

Comment:
Can't we set this value as default value? so we don't need to always pass it.

Reply:
we can, but I don't know if it's good to default values in use cases !